src/main/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/service/JiraDeploymentInfoSenderImpl.java

```diff
@@ -209,15 +209,15 @@ private Deployments createJiraDeploymentInfo(
 
     private JiraSendInfoResponse handleDeploymentApiResponse(
             final String jiraSite, final DeploymentApiResponse response) {
-        if (!response.getAcceptedDeployments().isEmpty()) {
+        if (response.getAcceptedDeployments() != null && !response.getAcceptedDeployments().isEmpty()) {
```

Comment:
This is the line that fixes #77, but I added the check to the others as well for good measure.